variables.tf

```diff
@@ -13,6 +13,12 @@ variable "allowed_account_ids_write" {
   type        = list(string)
 }
 
+variable "dest_region" {
+  default     = ""
+  description = "Region to replicate repo bucket to"
+  type        = string
+}
+
 variable "logging_bucket" {
```

Comment:
should we support a logging bucket for the replica?

Reply:
I figured it wasn't necessary but I can definitely add it.